lib/matplotlib/backends/backend_qt5.py

```diff
@@ -430,16 +430,17 @@ def flush_events(self):
         global qApp
         qApp.processEvents()
 
-    def start_event_loop(self, timeout):
-        FigureCanvasBase.start_event_loop_default(self, timeout)
-
-    start_event_loop.__doc__ = \
-                             FigureCanvasBase.start_event_loop_default.__doc__
-
-    def stop_event_loop(self):
-        FigureCanvasBase.stop_event_loop_default(self)
-
-    stop_event_loop.__doc__ = FigureCanvasBase.stop_event_loop_default.__doc__
+    def start_event_loop(self, timeout=0):
```

Comment:
This entire machinery is only used by blocking_input.py (ginput), in order to not block event handling while waiting for the user to select points on the plot.  See https://wiki.qt.io/Threads_Events_QObjects#Forcing_event_dispatching for the pattern.